invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -50,6 +57,25 @@ function InVideoQuizXBlock(runtime, element) {
         }
     }
 
+    function getVideoDimensions() {
+        videoPosition = $('.tc-wrapper', video).position().top;
```

Comment:
Nitpick: I tend to avoid prefixing variables if we're doing it for everything.
From the context of this function `getVideoDimensions`, it's obvious that
any dimensions we list are for videos, unless otherwise noted.

Not only does it cut down on some verbosity, but it also makes it easier to grep the code: grepping for `position` (case sensitive) won't find `videoPosition`, but it will find `position`.